web_timeline/README.rst

```diff
@@ -0,0 +1,221 @@
+| event_open_popup   | No        | When set to true, it allows to edit the events in a popup. If not (default value), the record is edited changing to form view.                                                                                                                                            |
++--------------------+-----------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| stack              | No        | When set to false, items will not be stacked on top of each other such that they do overlap.                                                                                                                                                                              |
++--------------------+-----------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| colors             | No        | Allows to set certain specific colors if the expressed condition (JS syntax) is met.                                                                                                                                                                                      |
++--------------------+-----------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| dependency_arrow   | No        | Set this attribute to a x2many field to draw arrows between the records referenced in the x2many field.                                                                                                                                                                   |
++--------------------+-----------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Optionally you can declare a custom template, which will be used to render the
+timeline items. You have to name the template 'timeline-item'.
+These are the variables available in template rendering:
+
+* ``record``: to access the fields values selected in the timeline definition.
+* ``field_utils``: used to format and parse values (see available functions in ``web.field_utils``).
+
+You also need to declare the view in an action window of the involved model.
+
+Example:
+
+.. code-block:: xml
+
+    <?xml version="1.0" encoding="utf-8"?>
```

Comment:
This code snippet isn't valid for this version. You can take the snippet from your un work :): https://github.com/OCA/project/blob/68f70f828c6a7a91f7e02f19389d4df7b5838f35/project_timeline/views/project_task_view.xml#L7-L62